main.go

```diff
@@ -45,6 +45,13 @@ func main() {
 			Name:  "namespace, n",
 			Usage: "Set namespace. Uses the namespace set in the context by default",
 		},
+		cli.StringSliceFlag{
+			Name: "selector, l",
+			Usage: `Only run using the releases that match labels. Labels can take the form of foo=bar or foo!=bar.
+	A release must match all labels in a group in order to be used. Multiple groups can be specified at once.
+	--selector tier=frontend,tier!=proxy --selector tier=backend. Will match all frontend, non-proxy releases AND all backend releases.
```

Comment:
should all references to selector be changed to `labels`?

Reply:
similar to k8s, the idea is that labels are on objects, and a selector selects based on labels

Reply:
ahh if that's the case the docs that are calling out `--label` should be changed right?

```--labels tier=frontend,tier!=proxy --labels tier=backend. Will match all frontend, non-proxy releases AND all backend releases.```

to 
```
--selector tier=frontend,tier!=proxy --selector tier=backend. Will match all frontend, non-proxy releases AND all backend releases.```

Reply:
Agreed we should use --selector and -l, but we just need to make sure all the docs align.

Reply:
it should be consistent across the docs now